### PR TITLE
Reverse title and time in timetable grid

### LINKED ISSUE #225: Reverse title and time in timetable grid
## Overview
The current implementation of the timetable grid displays the title and time in reverse order compared to the design. This needs to be corrected to match the design specifications.
image

## Links
- Design: https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI
- Code: https://github.com/DroidKaigi/conference-app-2024/blob/27ea36f1ae39693bbdb51c4d90e2a126551c8fc0/feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/TimetableScreen.kt#L91

### PR DESCRIPTION
**Pull request by TomoyaShibata**
## Issue
- close #225

## Overview (Required)
- Reverse title and time in timetable grid
  - also fixed the spacing with the title
- Side note
  - When I look at Figma, there is no space to only the left of the tilde, is this the correct design?
  - image

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
|

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

**Comment by takahirom** [comment]
This may be a little difficult to fix. But can we fix this..?
image

**Comment by takahirom** [comment]
Sorry. I merged another PR. But that could solve the issue 🙏

**Comment by TomoyaShibata** [comment]
Ok!
I'll try merge main branch, one more time check preview diff.

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridItem.kt [hunk 1]
```diff
@@ -305,8 +311,8 @@ private fun calculateFontSizeAndLineHeight(
     titleLength: Int,
 ): Pair<TextUnit, TextUnit> {
     // The height of the title that should be displayed.
-    val titleToScheduleSpaceHeightPx = with(localDensity) {
-        TimetableGridItemSizes.titleToSchedulePadding.toPx()
+    val scheduleToTitleSpaceHeightPx = with(localDensity) {
```
Comment (takahirom) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableGridItem.kt:314]: Thank you for adjusting this!

**Comment by takahirom** [comment]
@nobonobopurin 
Do we need space before the "~" or the design is correct?
image

**Review by takahirom — APPROVED** [review]
Anyway, this is a great contribution! Thank you!

**Comment by nobonobopurin** [comment]
@takahirom It appears to be spaced due to the font issue, but there is no actual space. If it looks like adjustments can be made visually, feel free to take care of that.

**Pull request merged**